### ISSUE TITLE
Understanding SC 1.4.12 Text Spacing - images not showing

### ISSUE THREAD

**Issue by kengdoj** [w3c/wcag#2564]
Figures 1 and 2 images are not displaying correctly on [Understanding SC 1.4.12 Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)

**Comment by patrickhlauke** [comment]
was about to say that i thought this was already fixed in https://github.com/w3c/wcag/pull/1549 ... but it seems that PR (which reintroduced the missing images_ still managed to introduce a different bug and drop a letter from the filenames.

made a follow-up that corrects this for good https://github.com/w3c/wcag/pull/2565

**Comment by kengdoj** [comment]
thanks @patrickhlauke for the speedy fix!
